libportal/portal-qt5.h

```diff
@@ -21,7 +21,85 @@
 
 #include <libportal/portal.h>
 
+#include <QMap>
+#include <QStringList>
+#include <QSharedPointer>
+#include <QVariant>
 #include <QWindow>
 
 XDP_PUBLIC
 XdpParent *xdp_parent_new_qt (QWindow *window);
+
+// Account portal helpers
+typedef struct {
+    QString id;
+    QString name;
+    QString image;
+} XdpQtGetUserInformationResult;
+
+XDP_PUBLIC
+XdpQtGetUserInformationResult xdp_qt_get_user_information_result_from_variant(GVariant *variant);
+
+// FileChooser portal helpers
+typedef enum {
+    XDP_QT_FILECHOOSER_FILTER_RULE_TYPE_PATTERN = 0,
+    XDP_QT_FILECHOOSER_FILTER_RULE_TYPE_MIMETYPE = 1
+} XdpQtFileChooserFilterRuleType;
+
+typedef struct {
+    XdpQtFileChooserFilterRuleType type;
+    QString rule;
+} XdpQtFileChooserFilterRule;
+
+typedef struct {
+    QString label;
+    QList<XdpQtFileChooserFilterRule> rules;
+} XdpQtFileChooserFilter;
+
+typedef struct {
+    QString id;
+    QString label;
+    QMap<QString, QString> options;
+    QString selected;
+} XdpQtFileChooserChoice;
+
+XDP_PUBLIC
+GVariant *xdp_qt_filechooser_files_to_variant(const QStringList &files);
```

Comment:
If these functions are to be called from Qt codebases, it could make sense to use camel-case.

Also, it could make sense to wrap this all in a namespace (this Qt wrappers have not been released yet, have they? otherwise we can't for ABI compatibility).